lib/ably/realtime/channel.rb

```diff
@@ -337,7 +337,7 @@ def history(options = {}, &callback)
             error = Ably::Exceptions::InvalidRequest.new('option :until_attach is invalid as the channel is not attached' )
             return Ably::Util::SafeDeferrable.new_and_fail_immediately(logger, error)
           end
-          options[:from_serial] = properties.attach_serial
+          options[:fromSerial] = properties.attach_serial
```

Comment:
> tests started failing recently

But _which_ tests? I’d like to understand which test failures motivated you to make this change?

Reply:
I understand that the line of code says `from_serial` and that the spec says `fromSerial`. But it's not automatically obvious that this is actually an issue — perhaps, for example, we have some code elsewhere that converts things from `snake_case` to `camelCase` 🤷. My point is, this code has existed for a long time, and as far as I can tell, the tests related to the functionality that it supports are passing, so if we're going to change it then I’d like to be sure that it's actually a problem.

Reply:
Actually, I did check and there is no code that converts `snake_case` to `camelCase`. I checked the logged HTTP `GET` request and it was sending pure `from_serial` as a param instead of `fromSerial`. 
The failing test is the same `skipped` test -> https://github.com/ably/ably-ruby/pull/391#discussion_r1613837639. If you run test locally, you will be able to see debug logs 

`
18:55:29.524 [debug] => URL: get https://sandbox-rest.ably.io/channels/persisted%3A616d/messages?direction=backwards&fromSerial=e025wxEqQBcQLz51419995%401716989128323-1&limit=100, Headers: Content-Type: "application/json", Accept: "application/json", User-Agent: "Ably Ruby client 1.2.5 ...
`
If I revert above change I get 
`
19:00:36.981 [debug] => URL: get https://sandbox-rest.ably.io/channels/persisted%3Aab9c/messages?direction=backwards&from_serial=e02kMFlRABcQOO73662964%401716989435972-1&limit=100, Headers: Content-Type: "application/x-msgpack", Accept: "application/x-msgpack", 
`
Also, none of other SDKs send `from_serial` as a param. It was surprising that ruby sends this param and still it works. There might be change implemented at server side to convert it into `fromSerial` but somehow I am not comfortable with it. Because, if things change in the future, `from_serial` will start failing for the ruby SDK and it also goes against spec.

Reply:
Let me know if I am missing anything :

Reply:
Also, as per https://ably.atlassian.net/browse/REA-1910 there are changes made related to `fromSerial`. So, I am skeptical if it works as of now. So, just trying to play safe and sending param mentioned in the spec.

Reply:
Seems, I will revert the change for now and will implement it ( if needed ) as a part of the working test itself.

Reply:
> Seems, I will revert the change for now and will implement it as a part of the working test itself.

Yeah, I think that what you've suggested makes sense. So you're saying you'll remove the `from_serial` -> `fromSerial` rename from this PR and instead do it in a PR that fixes the skipped test?

Reply:
Yes, thats right 👍